samples/tab-device-permissions/nodejs/src/components/CaptureImageDesktop.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { useEffect, useState } from 'react';
+import * as microsoftTeams from "@microsoft/teams-js";
+import { Card, Flex, Text, Button, CardHeader, CardBody, Image } from '@fluentui/react-northstar'
+
+/**
+ * The 'CaptureImageDesktop' component
+ * of your app.
+ */
+const CaptureImageDesktop = () => {
+    var imageCapture: ImageCapture;
+    const [capturedImage, setCapturedImage] = useState('');
+    const [capturedImages, setCapturedImages] = useState<any[]>([]);
+
+    useEffect(() => {
+        // initializing microsoft teams sdk
+        microsoftTeams.initialize()
+    })
+
+    // var imageCapture: ImageCapture;
+
+    // Method to validate before capturing media
+    function captureMedia() {
+        // Method to ask for image capture permission and then select media
+
+        navigator.permissions.query({ name: 'camera' }).then(function (result) {
+            if (result.state == 'denied') {
+                alert("failed");
```

Comment:
are we not getting the image here? why do we require video part again?

Reply:
We are taking a snapshot of the video for capturing image as there is no separate navigator method for capturing image. So video part is required here.